docs/quickstarts/nextjs.mdx

```diff
@@ -118,68 +120,119 @@ The [`<ClerkProvider>`](/docs/components/clerk-provider) component provides acti
-
-Add the `<UserButton />` anywhere inside `<ClerkProvider>` in your app. First, add `import { UserButton } from "@clerk/nextjs";` to the top of your file. Then, add `<UserButton afterSignOutUrl="/"/>`. The `afterSignOutUrl` prop lets you customize what page the user will be redirected to after sign out.
 
 <CodeBlockTabs type="router" options={["App Router", "Pages Router"]}>
-```tsx filename="app/page.tsx" {1,6}
-import { UserButton } from "@clerk/nextjs";
+```tsx filename="app/page.tsx" {1,6-11}
+import { SignedIn, SignedOut } from "@clerk/nextjs";
 
 export default function Home() {
   return (
     <div className="h-screen">
-      <UserButton afterSignOutUrl="/"/>
+      <SignedOut>
+        <p>This content is public. Only signed out users can see this.</p>
+      </SignedOut>
+      <SignedIn>
+        <p>This content is private. Only signed in users can see this.</p>
+      </SignedIn>
     </div>
   )
 }
 ```
 
-```tsx filename="pages/index.tsx" {1,7}
-import { UserButton } from "@clerk/nextjs";
+```tsx filename="pages/index.tsx" {1,6-11}
+import { SignedIn, SignedOut } from "@clerk/nextjs";
 
 export default function Home() {
   return (
     <>
-      <header>
-        <UserButton afterSignOutUrl="/" />
-      </header>
-      <div>Your home page's content can go here.</div>
+      <SignedOut>
+        <p>This content is public. Only signed out users can see this.</p>
+      </SignedOut>
+      <SignedIn>
+        <p>This content is private. Only signed in users can see this.</p>
+      </SignedIn>
     </>
   );
 }
+``` 
+</CodeBlockTabs>
+
+Try accessing your app now by visiting [`http://localhost:3000`](http://localhost:3000) to see what signed out users will see. Next you will add a button so you can sign in and see the content within the `<SignedIn />` component.
+
+
+### Embed the `<UserButton />` and `<SignInButton />`
+
+Clerk offers a set of [prebuilt components](/docs/components/overview) to add functionality to your app with minimal effort. The [`<UserButton />`](/docs/components/user/user-button) allows users to manage their account information and log out, completing the full authentication circle. The [`<SignInButton />`(/docs/components/unstyled/sign-in-button)] is an unstyled component designed to wrap around your own custom button. It can also act as a basic button on its own.
+
+
+To add the `<UserButton />` and `<SignInButton />` components, first add `import { UserButton, SignedInButton } from "@clerk/nextjs";` to the top of your file. Then, add `<UserButton afterSignOutUrl="/"/>` inside of the `<SignedIn />` components. The `afterSignOutUrl` prop lets you customize what page the user will be redirected to after sign out. Finally, add `<SignInButton />` component inside of the `<SignedOut /> components`
+

```

Comment:
"Clerk offers a set of prebuilt components" in every section that mentions a component is quite repetitive. We can omit this.

Also, we can 
1. reorganize the information in the order they are presented in order to improve flow and clarity (in the code example, the `<SignInButton />` comes first, and then the addition of the `<UserButton />`.
2. reword how we are explaining the code example. we don't need to tell them step by step what they need to do programmatically; they can see this in the code example. instead, we can give an *explanation* of what is being added, and *why* those additions are being made.

```suggestion
### Embed the `SignInButton />` and `<UserButton />`

The [`<SignInButton />`](/docs/components/unstyled/sign-in-button)] is an unstyled component designed to wrap around your own custom button. It can also act as a basic button on its own. The [`<UserButton />`](/docs/components/user/user-button) allows users to manage their account information and sign out, completing the full authentication circle.

In the following example, the `<SignInButton />` is added to the `<SignedOut />` component to allow signed-out users to sign in. The `<UserButton />` is added to the `<SignedIn />` component to allow signed-in users to manage their account, add an account, or sign out. 
```

Reply:
"reword how we are explaining the code example. we don't need to tell them step by step what they need to do programmatically; they can see this in the code example. instead, we can give an explanation of what is being added, and why those additions are being made."

The changes here follow the precedent set in the Next QS rewrite and in the React QS rewrite. I think shipping this version works short term and if this is the preferred change it can be added later.